standalone/app/controllers/OAuthLoginController.scala

```diff
@@ -57,13 +57,20 @@ object OAuthLoginController extends Controller with ExecutionContexts with impli
     }
   }
 
+  private def overrideRedirectUrl(config: GoogleAuthConfig)(implicit request: RequestHeader) = {
+    request.getQueryString("redirect-url") match {
+      case Some(url) => config.copy(redirectUrl = url)
```

Comment:
In other tools we validated the domain of the redirect url argument. Should we do this here (and everywhere as a rule) for security?

cc @adamnfish